.github/ISSUE_TEMPLATE/broken_addon.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Report a broken add-on
+about: Report an add-on that is currently broken within Kodi.  
+title: ''
+labels: 'Broken-Addon'
+assignees: ''
+
+---
+
+
+### Add-on details:
+<!--- Provide some details of te broken add-on. -->
```

Comment:
"the" broken add-on .. same error in the other template too

Reply:
Thanks! I read it and read it, but still.